orderer/common/cluster/util_test.go

```diff
@@ -1474,3 +1476,61 @@ func sampleConfigBlock() *common.Block {
 		},
 	}
 }
+
+func TestGetTLSSessionBinding(t *testing.T) {
+	serverCert, err := ca.NewServerCertKeyPair("127.0.0.1")
+	require.NoError(t, err)
+
+	clientCert, err := ca.NewClientCertKeyPair()
+	require.NoError(t, err)
+	srv, err := comm.NewGRPCServer("127.0.0.1:0", comm.ServerConfig{
+		SecOpts: comm.SecureOptions{
+			Certificate:       serverCert.Cert,
+			Key:               serverCert.Key,
+			UseTLS:            true,
+			ClientRootCAs:     [][]byte{ca.CertBytes()},
+			RequireClientCert: true,
+		},
+	})
+	require.NoError(t, err)
+
+	dispatcher := &mocks.Dispatcher{}
+	dispatcher.On("DispatchConsensus", mock.Anything, mock.Anything).Return(nil)
```

Comment:
in this test, you create the stream. You don't send anything to the stream, so you don't need this dispatcher.